src/Combobox/index.stories.js

```diff
@@ -237,14 +238,17 @@ CustomFiltering.parameters = {
   },
 };
 
-export const WithActions = (args) => {
+export const WithActions = () => {
   const [isOpen, setIsOpen] = useState(false);
+  const [inputValue, setInputValue] = useState("");
```

Comment:
Change example to a controlled component, which is the norm for us and expected by `ContextForm` and DBBL form adapters.